lazyflow/utility/orderedSignal.py

```diff
@@ -22,6 +20,9 @@
 # 		   http://ilastik.org/license/
 ###############################################################################
 from collections import OrderedDict
+import logging
+
+logger = logging.getLogger("lazyflow.signals")
```

Comment:
using this naming convention here is probably fine as `lazyflow.signals` will be findable.